pilot/lib/pilot/tester_manager.rb

```diff
@@ -17,17 +17,13 @@ def add_tester(options)
         app: app
       )
       begin
+        # Groups are now required
         groups = Spaceship::TestFlight::Group.add_tester_to_groups!(tester: tester, app: app, groups: config[:groups])
-        if tester.kind_of?(Spaceship::Tunes::Tester::Internal)
-          UI.success("Successfully added tester to app #{app.name}")
+        if config[:groups]
```

Comment:
Groups is kind of a required param now right? Not sure if we should change this to just handle the groups param, and make the param itself required? Although making it required I dont think will work because `add` is not the default command so it wouldnt make sense to require it on uploading builds to TestFlight.

Maybe should check for groups right after `begin` and if there arent any, throw an exception saying that its required.

Reply:
Gread idea